HHS/Head-Start-TTADP
# [TTAHUB-270] Remove old approving manager code

## Description of change
- Remove the columns "oldApprovingManagerId" & "oldManagerNotes" from the ActivityReports table.
- Remove the no-longer needed transitionToMultiApprover scripts, as the tests for these contained numerous references to the abandoned field.
- Remove additional references to those fields across the app

## How to test
Ensure site still works as normal, confirm that columns in question have been removed from the DB and also if possible confirm that the docs and app code contain no more references to these columns.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-270

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Boundary diagram updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
